src/analysis/souffle/tests/dp_sensitivity_tests/expectedResult.facts

```diff
@@ -0,0 +1,5 @@
+revenue_table_where1;1
+revenue_table_group_by;2
+revenue_table_where2;1
+user;2
+address;2
```

Comment:
End file with newline. If you're using vscode, it should be possible to configure it to always put a newline at the end of your files.